neo/SmartContract/InteropService.cs

```diff
@@ -18,6 +18,9 @@ namespace Neo.SmartContract
 {
     public static partial class InteropService
     {
+        public const int MaxStorageKeySize = 512;
+        public const int MaxStorageValueSize = 65536;
+
```

Comment:
We could be even more strict on key size. Some platforms adopt 64 bytes for key, as this would directly affect a MPT implementation. I don't think this applies as well to Sparse Merkle Trees, that I'm implementing right now, but this field should be the smallest possible. Value is more flexible I think.